docs/eloquent-models/relationships.txt

```diff
@@ -0,0 +1,514 @@
+============================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: php framework, odm, code example, entity relationship, eloquent
+
+Overview
+--------
+
+This page describes the following Laravel Eloquent and MongoDB-specific
+relationships available in {+odm-short+} and shows examples of how to define
+and use them:
+
+- :ref:`One to one relationship <laravel-eloquent-relationship-one-to-one>`,
+  created by using the ``hasOne()`` method and its inverse, ``belongsTo()``
+- :ref:`One to many relationship <laravel-eloquent-relationship-one-to-many>`,
+  created by using the ``hasMany()`` and its inverse, ``belongsTo()``
+- :ref:`Many to many relationship <laravel-eloquent-relationship-many-to-many>`,
+  created by using the ``belongsToMany()`` method
+- :ref:`Embedded document pattern <laravel-embedded-document-pattern>`, a
+  MongoDB-specific relationship that can represent a one to one or one to many
+  relationship, created by using the ``embedsOne()`` or ``embedsMany()`` method
+- :ref:`Cross-database relationships <laravel-relationship-cross-database>`,
+  required when you want to create relationships between MongoDB and SQL models
+
+Add a function to the model class that calls the appropriate relationship method
+to establish a relationship. This function allows you to access the related
+model as a **dynamic property**. A dynamic property lets you access the
+related model by using the same syntax as you use to access a property on the
+model.
+
+.. _laravel-eloquent-relationship-one-to-one:
+
+One to One Relationship
+-----------------------
+
+A one to one relationship between models consists of a model record related to
+exactly one other type of model record. In MongoDB, a record is represented as
+a document, and different model types exist in separate collections.
+
+When you add a one to one relationship by using the method, Eloquent lets you
+access the model by using a dynamic property and stores the model's document
+ID on the related model.
+
+In {+odm-short+}, you can define a one to one relationship by using the
+``hasOne()`` method or ``belongsTo()`` method.
+
+When you add the inverse of the relationship by using the ``belongsTo()``
+method, Eloquent lets you access the model by using a dynamic property, but
+does not add any fields.
+
+To learn more about one to one relationships, see
+`One to One <https://laravel.com/docs/{+laravel-docs-version+}/eloquent-relationships#one-to-one>`__
+in the Laravel docs.
+
+One to One Example
+~~~~~~~~~~~~~~~~~~
+
+The following example class shows how to define a ``HasOne`` one to one
+relationship between a ``Planet`` and ``Orbit`` model.
+
+.. literalinclude:: /includes/eloquent-models/relationships/PlanetOneToOne.php
+   :language: php
+   :dedent:
+
+The following example class uses a dynamic property and calls the
+``belongsTo()`` method to define the inverse of the relationship on ``Orbit``
+as shown in the following example class:
```

Comment:
S: fix that the first clause and last clause are redundant
```suggestion
The following example class uses a dynamic property and calls the
``belongsTo()`` method to define the inverse of the relationship on ``Orbit``:
```

Reply:
I'll rephrase to fix some of the information that I forgot to update.